docs/asyncapi.yaml

```diff
@@ -260,4 +187,166 @@ components:
         value:
           type: string
           description: The reduction value
-          example: 75.00
+          example: '75.00'
+    FundingTotal:
+      type: object
+      required:
+      - name
+      - area
+      - annualValue
+      - quarterlyValue
+      - quarterlyReduction
+      - quarterlyPayment
+      properties:
+        name:
+          $ref: '#/components/schemas/FundingTotalName'
+        level:
+          $ref: '#/components/schemas/FundingTotalLevel'
+        area:
+          $ref: '#/components/schemas/FundingArea'
+        rate:
+          $ref: '#/components/schemas/FundingTotalRate'
+        annualValue:
+          $ref: '#/components/schemas/FundingAnnualValue'
+        quarterlyValue:
+          $ref: '#/components/schemas/FundingQuarterlyValue'
+        quarterlyReduction:
+          $ref: '#/components/schemas/FundingQuarterlyReduction'
+        quarterlyPayment:
+          $ref: '#/components/schemas/FundingQuarterlyPayment'
+    FundingName:
+      type: string
+      description: The funding option name
+      example: Arable and horticultural soils
+    FundingTotalName:
+      type: string
+      description: The funding option name
```

Comment:
Description should not be the same as `FundingName` above, "The funding total name"